cfme/configure/settings.py

```diff
@@ -349,12 +349,17 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
 
     @classmethod
-    def set_default_view(cls, button_group_name, default):
+    def save(cls):
```

Comment:
**Required**
Is the intent here to speed up the default view setting in the test when you're changing multiple views?

If so, I'm pretty sure I'd rather see this class take a list of button_group_names. Should be minor to refactor existing uses to pass a list, or to handle both strings and lists here.